x-pack/plugins/lists/server/saved_objects/exception_list.ts

```diff
@@ -47,10 +47,20 @@ export const commonMapping: SavedObjectsType['mappings'] = {
       type: 'keyword',
     },
     name: {
-      type: 'keyword',
+      fields: {
+        keyword: {
+          type: 'keyword',
+        },
+      },
+      type: 'text',
```

Comment:
We need to be careful here so we don't introduce a bigger inconsistency than we already have with which comes first, the `.text` or the `.keyword` and which is the default?

For references:
https://github.com/elastic/ecs/blob/master/generated/elasticsearch/7/template.json#L171-L179
https://github.com/elastic/elasticsearch/issues/53181

ECS looks to be `xyz.text` and _not_ the other way which is `xyz.keyword`. I am thinking you want to flip this around to be keyword first by default (existing behavior as well) and then if they want `.text` they have to explicitly add `.text`